webapp/app/js/services/main.js

```diff
@@ -215,3 +215,20 @@ treeherder.factory('thNotify', [
     return thNotify;
 
 }]);
+
+treeherder.factory('thPlatformName', [
```

Comment:
This **was** only in ``clonejobs.js`` but needed to be shared elsewhere.  Namely ``plugins controller.js``.